src/parse.js

```diff
@@ -20,24 +20,30 @@ parse.section = function(conf, section) {
 	let sectionUrl;
 	let newTab;
 
+	let collection = section.collection;
+
 	if (section['external-url'] !== undefined) {
 		sectionUrl = section['external-url'].href;
 		newTab =
 			section['external-url']['new-tab'] !== undefined ? section['external-url']['new-tab'] : true;
+		// Use title for collection if it's not set
+		if (collection === undefined) {
+		  collection = section.title
+    }
 	} else {
 		sectionUrl = path.join(conf.baseUrl, section.url);
 		newTab = false;
 	}
 
-	const sectionPath = path.join(conf.contentPath, `_${section.collection}`, '/');
+	const sectionPath = path.join(conf.contentPath, `_${collection}`, '/');
 
     return {
         id: sectionPath,
         type: structure.TYPE.SECTION,
         title: section.title,
         path: sectionPath,
         url: sectionUrl,
-        collection: section.collection,
+        collection: collection,
```

Comment:
Since we're using ES6, you can just go `collection,` (instead of `collection: collection`), since the key and value are exactly the same.

Reply:
this is cool. i didn't know javascript support this.